test/unit/machine-learning/machine-learning-api-client.spec.ts

```diff
@@ -486,6 +578,32 @@ describe('MachineLearningApiClient', () => {
         });
     });
 
+    it('should resolve properly with tflite managed model', () => {
```

Comment:
Drop "properly". Here and below. Also give more context "should resolve with Model for tflite managed model response"

Reply:
done